### ISSUE TITLE
Webpack compiling template literals `s to "s or 's -- causing translation issues

### ISSUE THREAD

**Issue by dcress** [webpack/webpack-cli#1618]
First off, thank you so much for Webpack! It's so helpful!

### The Bug
I'm running into an issue that I can't seem to find the solution to via Google, SO or GH. My company's website is translated into 4 different languages. We use a in-house built translator to accomplish this task. For our JS/TS files, we use a nuggeting system `[[[Translate text]]]` which replaces the text with what our international brand managers have inputted into our CMS. This is done AFTER we compile with webpack (it's a part of our deployment process). 99% of the time we don't have an issue, but on occasion because of the way webpack compiles our template literal's backtics to quotes we run into an issue. Let me give an example: 

**Working example:**
Input:
```ts
let errMessage = `[[[Oh no! Something went wrong!]]];
```
Output (translated):
```js
let errMessage = "¡Oh no! Algo salió mal."
```

***Breaking Example***
Input:
```ts
let errMessage = `[[[Your something has not been saved -- please "Save to Cloud" to save your changes.]]];
```

Output (translated):
```js
let errMessage = 'Votre chose n'a pas été enregistrée - veuillez "Enregistrer dans le cloud" pour enregistrer vos modifications.'
```

As you can see, because the Nugget contained "s, webpack switched the backtics to 's. However, in French our translation uses `n'a`, which breaks the `` string and caused our JS file to break. 

### My Request/Possible Fix
I have yet to find a way to tell Webpack to preserve the template literals backtics. If there's a configuration option that I should be using, please let me know, but nothing has come up in my research. Template literals are pretty well supported now, so I don't fully understand why they're being compiled to "s or 's. 

### TLDR;
Is there a way to manually tell Webpack to not compile template literals backtics to "s or 's?

**Comment by evilebottnawi** [comment]
Can you provide example of `s` literals?

**Comment by dcress** [comment]
Hey @evilebottnawi! Maybe I'm misunderstanding your question, but I have a couple examples in the above bug description. Generally template literals (or string interpolation) will look like so: 

```
let errMessage = `This is my message ${someOtherVar} blah blah blah...`;
```

However, if you have dynamic content that will be coming into your code (via out nugget system, for instance) we tend to use backticks (```) instead of "s or 's, since we're not 100% sure what our translator will put in his translation.

**Comment by evilebottnawi** [comment]
How it is related to webpack-cli?

**Comment by DanielRuf** [comment]
> As you can see, because the Nugget contained "s, webpack switched the backtics to 's

For me it looks like the issue is about double and single quotes. Normally they should be escaped or htmlentities used maybe.

**Comment by dcress** [comment]
@evilebottnawi  @DanielRuf Thanks for the replies! Yes, it's related to webpack because webpack is changing my backticks to either "s or 's. I don't know why Webpack would be opinionated in this way as backticks/template literals are [well supported](https://caniuse.com/#search=template%20literals) in all modern browsers. 

Again, it could be that I'm missing something here, but it seems like something that webpack should just allow to pass through (as in, not change), or at the very least allow a config setting to control that behavior. 

@DanielRuf I would normally chalk it up to needing to escape characters, but in our particular case we're translating these strings, so we can't really ask our translators (who aren't developers) to add in escapes. 

As I mentioned above, the main thing for me is: Why does Webpack have this behavior? And is there a way to prevent it?

**Comment by DanielRuf** [comment]
It may depend on the babel / webpack target settings (which ES version and features) so it applies the transformations.

**Comment by evilebottnawi** [comment]
Yes, we don't change your code, you can create reproducible test repo and I will say which loader/plugin break it

**Comment by dcress** [comment]
@evilebottnawi @DanielRuf Ah, okay that's great to know. I'll take a closer look there and see if i can parse out which loader is causing the issue. I was assuming it was a core webpack-cli issue. Thanks for the help, y'all!

**Comment by kipid** [comment]
I have the exactly same problem now.
Since backtick (``) can use `${someVar} is something.` and including explicit enters `some
value
is
here`.
Forcing converting this string to single quote ('') or double quote ("") makes problem to me.
Since I supporting multi-language by pattern-replacer JAVA code which I made. This problem makes me crazy.

Copilot (ChatGPT) says do
{
  "presets": [
    ["@babel/preset-env", {
      "targets": ">0.25%, not dead"
    }],
    "@babel/preset-react"
  ]
}
setting. I am not sure this make my problems solved. I will try.
